### PR TITLE
Etoile

### PR DESCRIPTION
**Pull request by AsadPug**
Check commits for more information. (debrouille toi jsuis parresseuse)

### CODE REVIEW

File: Orion_client/controleur_serveur.py
Comment (AsadPug) [inline Orion_client/controleur_serveur.py:1]: planete -> etoile

File: Orion_client/gestionnaire_vue.py
Comment (AsadPug) [inline Orion_client/gestionnaire_vue.py:1]: planete -> etoile

File: Orion_client/modeles/batiment.py
Comment (AsadPug) [inline Orion_client/modeles/batiment.py:1]: planete -> etoile

File: Orion_client/modeles/etoile.py
Comment (AsadPug) [inline Orion_client/modeles/etoile.py:1]: planete -> etoile

File: Orion_client/modeles/vaisseau.py
Comment (AsadPug) [inline Orion_client/modeles/vaisseau.py:1]: planete -> etoile

File: Orion_client/orion_modele.py
Comment (AsadPug) [inline Orion_client/orion_modele.py:1]: planete -> etoile

File: Orion_client/test.py
Comment (AsadPug) [inline Orion_client/test.py:1]: planete -> etoile

**Review by AsadPug — COMMENTED** [review]
planete -> etoile

**Review by Thelazerlog — APPROVED** [review]

**Pull request merged**